3D Tank/Assets/Scripts/Bullet/BulletView.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletView : MonoBehaviour
```

Comment:
It should have all input as well as we pass all things from this class to controller .